src/Functions/array/FunctionArrayMapped.h

```diff
@@ -355,15 +355,15 @@ class FunctionArrayMapped : public IFunction
                     {
                         arrays.emplace_back(
                             column_tuple->getColumnPtr(j),
-                            recursiveRemoveLowCardinality(type_tuple.getElement(j)),
+                            type_tuple.getElement(j),
```

Comment:
I didn't understand why we remove LowCardinality recursively here from data type only, so the column may remain LowCardinality. Don't see why we need to remove LowCardinality here at all, let's check if some tests will fail